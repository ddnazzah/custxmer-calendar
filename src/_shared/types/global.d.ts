/*
 * @Author: Dieu-Donne Nazzah
 * @Date: 2021-03-20 03:02:21
 * @Last Modified by: Dieu-Donne Nazzah
 * @Last Modified time: 2021-10-19 20:33:19
 */

declare module 'accordion-collapse-react-native';
declare module 'react-native-sliders';
declare module 'react-native-global-font';
declare module 'rn-snackbar';
declare module 'react-native-smooth-pincode-input';

interface IFlex {
    dir?: 'column' | 'column-reverse' | 'row' | 'row-reverse';
    align?:
        | 'center'
        | 'end'
        | 'flex-end'
        | 'flex-start'
        | 'self-end'
        | 'self-start'
        | 'start';
    self?:
        | 'center'
        | 'end'
        | 'flex-end'
        | 'flex-start'
        | 'self-end'
        | 'self-start'
        | 'start';
    justify?:
        | 'space-around'
        | 'space-between'
        | 'space-evenly'
        | 'stretch'
        | 'center'
        | 'end'
        | 'flex-end'
        | 'flex-start'
        | 'start';
    flexWrap?: 'nowrap' | 'wrap' | 'wrap-reverse';
    flexShrink?: number;
    flex?:
        | 'auto'
        | 'content'
        | 'fit-content'
        | 'max-content'
        | 'min-content'
        | 'none'
        | (string & {})
        | (number & {});
    mx?: string | number;
    my?: string | number;
    mt?: string | number;
    mr?: string | number;
    ml?: string | number;
    mb?: string | number;
    m?: string | number;
    px?: string | number;
    py?: string | number;
    pt?: string | number;
    pr?: string | number;
    pl?: string | number;
    pb?: string | number;
    p?: string | number;
}

declare module '*.svg' {
    import React from 'react';
    import { SvgProps } from 'react-native-svg';
    const content: React.FC<SvgProps>;
    export default content;
}
