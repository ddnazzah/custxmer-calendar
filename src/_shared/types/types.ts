/*
 * @Author: Dieu-Donne Nazzah
 * @Date: 2021-04-15 17:07:48
 * @Last Modified by: Dieu-Donne Nazzah
 * @Last Modified time: 2022-10-25 15:22:03
 */

import 'styled-components';

export interface Theme {
    BACKGROUND: string;
    SURFACE: string;
    PRIMARY: string;
    ON_BACKGROUND: string;
    ON_SURFACE: string;
    ON_PRIMARY: string;
}

export type DatesProps = Array<{
    date: number;
    mood: 'happy' | 'fair' | 'sad';
}>;

declare module 'styled-components' {
    export interface DefaultTheme extends Theme {}
}
