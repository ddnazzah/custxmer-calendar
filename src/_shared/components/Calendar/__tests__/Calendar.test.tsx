/*
 * @Author: Dieu-Donne Nazzah
 * @Date: 2021-06-28 04:19:16
 * @Last Modified by: Dieu-Donne Nazzah
 * @Last Modified time: 2022-10-27 20:21:35
 */

import { render, screen } from '@testing-library/react-native';
import React from 'react';
import { Calendar } from '..';
import TestProvider from '../../../../@testing/Provider';
import { DatesProps } from '../../../types';

describe('<Calendar/>', () => {
    it('renders correctly', () => {
        const props = {
            dates: [
                {
                    date: 4,
                    mood: 'sad',
                },
                {
                    date: 1,
                    mood: 'sad',
                },
                {
                    date: 4,
                    mood: 'happy',
                },
                {
                    date: 6,
                    mood: 'fair',
                },
                {
                    date: 27,
                    mood: 'sad',
                },
            ] as DatesProps,
            month: 9,
            year: 2022,
        };

        render(
            <TestProvider>
                <Calendar calendarProps={props} />
            </TestProvider>,
        );

        expect(screen.toJSON()).toMatchSnapshot();
        expect(screen.getByText('September, 2022')).toBeDefined();
        expect(screen.getByText('30')).toBeDefined();
    });

    it('renders moods correctly', () => {
        const props = {
            dates: [
                {
                    date: 4,
                    mood: 'sad',
                },
                {
                    date: 1,
                    mood: 'sad',
                },
                {
                    date: 4,
                    mood: 'happy',
                },
                {
                    date: 6,
                    mood: 'fair',
                },
                {
                    date: 27,
                    mood: 'sad',
                },
            ] as DatesProps,
            month: 9,
            year: 2022,
        };

        render(
            <TestProvider>
                <Calendar calendarProps={props} />
            </TestProvider>,
        );

        expect(screen.getAllByTestId('happyMood')).toHaveLength(1);
        expect(screen.getAllByTestId('fairMood')).toHaveLength(1);
        expect(screen.getAllByTestId('sadMood')).toHaveLength(2);
    });
});
