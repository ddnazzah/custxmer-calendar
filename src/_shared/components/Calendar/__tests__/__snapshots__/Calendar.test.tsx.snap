// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Calendar/> renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
      },
    ]
  }
>
  <View
    accessible={true}
    collapsable={false}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#ffffff",
            "fontFamily": "Sora-Bold",
            "fontSize": 30,
          },
        ]
      }
    >
      September, 2022
    </Text>
  </View>
  <View
    style={
      Array [
        Object {
          "flexDirection": "row",
          "justifyContent": "space-between",
          "marginVertical": 20,
        },
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#ffffff",
            "fontFamily": "Sora-Medium",
            "fontSize": 18,
          },
        ]
      }
    >
      Mon
    </Text>
    <Text
      style={
        Array [
          Object {
            "color": "#ffffff",
            "fontFamily": "Sora-Medium",
            "fontSize": 18,
          },
        ]
      }
    >
      Tue
    </Text>
    <Text
      style={
        Array [
          Object {
            "color": "#ffffff",
            "fontFamily": "Sora-Medium",
            "fontSize": 18,
          },
        ]
      }
    >
      Wed
    </Text>
    <Text
      style={
        Array [
          Object {
            "color": "#ffffff",
            "fontFamily": "Sora-Medium",
            "fontSize": 18,
          },
        ]
      }
    >
      Thu
    </Text>
    <Text
      style={
        Array [
          Object {
            "color": "#ffffff",
            "fontFamily": "Sora-Medium",
            "fontSize": 18,
          },
        ]
      }
    >
      Fri
    </Text>
    <Text
      style={
        Array [
          Object {
            "color": "#ffffff",
            "fontFamily": "Sora-Medium",
            "fontSize": 18,
          },
        ]
      }
    >
      Sat
    </Text>
    <Text
      style={
        Array [
          Object {
            "color": "#ffffff",
            "fontFamily": "Sora-Medium",
            "fontSize": 18,
          },
        ]
      }
    >
      Sun
    </Text>
  </View>
  <View
    style={
      Array [
        Object {
          "flexDirection": "row",
          "flexWrap": "wrap",
        },
      ]
    }
  >
    <View
      shiftIndex={4}
      style={
        Array [
          Object {
            "height": 80,
            "width": 304.2857142857143,
          },
        ]
      }
    />
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <SvgMock
        testID="sadMood"
      />
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        1
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        2
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        3
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <SvgMock
        testID="happyMood"
      />
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        4
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        5
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <SvgMock
        testID="fairMood"
      />
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        6
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        7
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        8
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        9
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        10
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        11
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        12
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        13
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        14
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        15
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        16
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        17
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        18
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        19
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        20
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        21
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        22
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        23
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        24
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        25
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        26
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <SvgMock
        testID="sadMood"
      />
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        27
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        28
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        29
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "height": 50,
          "justifyContent": "flex-end",
          "marginTop": 30,
          "opacity": 1,
          "width": 101.42857142857143,
        }
      }
    >
      <Text
        _isToday={false}
        style={
          Array [
            Object {
              "alignSelf": "center",
              "color": "#ffffff",
              "fontFamily": "sora",
              "fontSize": 18,
              "fontWeight": "normal",
              "textAlign": "center",
            },
          ]
        }
      >
        30
      </Text>
    </View>
  </View>
</View>
`;
